unix/vncserver/vncsession.man.in

```diff
@@ -16,6 +17,14 @@ appropriate options and starts a window manager on the VNC desktop.
 is rarely called directly and is normally started by the system service
 manager.
 
+.SH -D OPTION
+.B vncsession
+by default forks and detaches so it normally is a systemd Type=forking service.
+If the -D option is used, it does not fork and detach. This option is provided
+for use with other types of systemd services such as Type=simple and Type=exec
+and for use with non-systemd system service managers that are not compatible
+with Type=forking services.
```

Comment:
This description gets a bit systemd-specific. I think it's sufficient to just state that it doesn't fork and detach to a daemon for scenarios where you want to keep it in the foreground.

Reply:
AFAICS, systemd is the only officially supported way in the TigerVNC project to manage vncsession, so I specifically mentioned systemd in the man page. But I am willing to remove the reference to systemd here so I think there is no problem here either.

Maybe in the future Void users or developers can contribute runit service files upstream to TigerVNC, just as maybe in the future Devuan users or developers might contribute a SysV init script upstream to TigerVNC, etc., to supplement the systemd unit that TigerVNC now ships. In that way, TigerVNC could at least officially provide some support for these alternate process supervision systems even if the support level might not be as high as users of systemd would get directly from TigerVNC. Of course this increases the maintenance burden a bit but that burden would be shared among other projects that use these alternate systems so I don't think it would be so bad.

I will work on these changes and I hope to have an updated PR by the end of the week after I carefully test it. In the meantime, hopefully the Void developers will explain in #1649 more clearly to what extent they consider running in the foreground a requirement for services in their runit process supervision system.

Reply:
I think recent comments at #1649 indicate some guidance about the purpose of the new option is indicated. IIUC, the option is specifically added for runit compatibility and is not useful for debugging vncsession from a user session on a terminal. So should we say so explicitly in the updated man page?

Reply:
I think that sounds appropriate, yes.